### PR TITLE
Feature/php 8 typehints

### PR DESCRIPTION
**Pull request by thomasnordahl-dk**

### CODE REVIEW

**Review by JyskFynskeMedierJoej — APPROVED** [review]

File: src/Tokenizer.php [hunk 1]
```diff
@@ -100,61 +95,73 @@ protected function statement()
...
-
-            $tokens[] = $token;
+            $token = $this->token();
         }
 
         return $tokens;
     }
 
     /**
+     * TODO: Refactor this - cyclomatic complexity > 10
```
Comment (boan-jfm) [inline src/Tokenizer.php:108]: Do we have a task on this?

File: src/Tokenizer.php [hunk 2]
```diff
@@ -250,9 +253,11 @@ protected function grouped()
     /**
      * @return string|null
```
Comment (boan-jfm) [inline src/Tokenizer.php]: Some places doc-blocks has been removed - I think this could be removed too

**Review by boan-jfm — APPROVED** [review]

File: src/Tokenizer.php (hunk 1 above)
Comment (thomasnordahl-dk) [reply src/Tokenizer.php:108]: I've added a jira ticket. I will send you a link via Teams :+1:

File: src/Tokenizer.php (hunk 2 above)
Comment (thomasnordahl-dk) [reply src/Tokenizer.php]: I went through the docblocks and removed those that didn't add any new information :+1:

**Pull request merged**